sql/core/src/test/scala/org/apache/spark/sql/FileBasedDataSourceSuite.scala

```diff
@@ -826,27 +826,6 @@ class FileBasedDataSourceSuite extends QueryTest
     }
   }
 
-  test("File table location should include both values of option `path` and `paths`") {
```

Comment:
@cloud-fan This is a test that's failing. Looks like this was a supported scenario and is it OK to deprecate this behavior?

Reply:
It's for file source v2 only, I think it's OK to drop it.